services/lab-service/lab-service/main.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 Telstra Open Source
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+import os
+import sys
+import time
+import signal
+import logging
+
+import requests
+from topology import Topology
+from lockkeeper import run_server
+
+logger = logging.getLogger()
+
+LAB_ID = os.environ.get("LAB_ID", 1)
+API_HOST = os.environ.get("API_HOST", 'lab-api.pendev:8288')
+
+
+if __name__ == '__main__':
```

Comment:
Can we produce normal entry points using setuptools?

nevermind, just crying...